mite_http/__init__.py

```diff
@@ -8,12 +8,27 @@
 logger = logging.getLogger(__name__)
 
 
-@asynccontextmanager
-async def _session_pool_context_manager(session_pool, context):
-    context.http = await session_pool._checkout(context)
-    yield
-    await session_pool._checkin(context.http)
-    del context.http
+class AcurlSessionWrapper:
+    def __init__(self, session):
+        self.__session = session
+        self.__callback = None
+        self.additional_metrics = {}
+
+    def getattr(self, attrname):
+        try:
+            r = object.___getattr__(self, attrname)
```

Comment:
```suggestion
            r = object.__getattr__(self, attrname)
```
Is this an eye test? 😝